### DISCUSSION TITLE
Size of "Original size" All archives

### DISCUSSION THREAD

**Discussion by izyk** [borgbackup/borg#5935] [Q&A] [answered]
Hello.
Why is the original size ("Original size" column) in the row describing all archives not equal to the sum of all archives added to the repository? Why does it show a different size that exceeds the sum of the original sizes of all archives?  What does this size show? 
"Deduplicated size" is true value. And repo is good. I can restore data from it.

```
------------------------------------------------------------------------------
                       Original size      Compressed size    Deduplicated size
This archive:                2.27 TB            467.83 GB             12.09 GB
All archives:               16.74 TB              3.16 TB            393.70 GB

```
Really sum of all my archives is 2.27*5=11,35 TB
I try check with --repair it is not help.

I create archives with this command:
/usr/bin/borg create --stats --read-special --chunker-params=12,23,17,4095 .....
and after:
borg prune ${pathtorepo} -v --list --keep-last=5

Does anyone have any thoughts on this? Or has anyone encountered this behavior? I have some repo on the servers and all of them have wrong Original size. All archives have virtual machines images as  --read-special.
Thank you.

**Comment by ThomasWaldmann** [comment, marked as answer]
I remember addressing some "stats related" issues in master branch, so maybe search for closed tickets on github issue tracker that are related to your issue. Some of the fixes needed quite big code changes, so it was not advisable to port them to 1.1-maint branch so they get into 1.1.x releases.

In general, sizes of something are hard to deal with in borg due to:
- deduplication
- checkpoint archives vs. normal archives
